packages/manager/apps/pci-load-balancer/src/pages/detail/pools/detail/members/edit/Edit.page.tsx

```diff
@@ -42,22 +39,29 @@ export default function EditPage() {
     onError(error: ApiError) {
       addError(
         <Translation ns="load-balancer">
-          {(_t) =>
-            _t('octavia_load_balancer_global_error', {
-              message: error?.response?.data?.message || error?.message || null,
-              requestId: error?.config?.headers['X-OVH-MANAGER-REQUEST-ID'],
-            })
-          }
+          {(_t) => (
+            <span
+              dangerouslySetInnerHTML={{
+                __html: _t('octavia_load_balancer_global_error', {
+                  message: (error?.response?.data as { message: string })
```

Comment:
maybe use {error:  { response : { data, message } } above to avoid long lines and repeating error.response while you are using response instead